unit/src/main/java/org/apache/falcon/unit/FalconUnitClient.java

```diff
@@ -285,14 +285,14 @@ public APIResult submitExtensionJob(String extensionName, String jobName, String
 
         InputStream configStream = getServletInputStream(configPath);
         try {
-            Map<EntityType, List<Entity>> entityMap = getEntityTypeListMap(extensionName, jobName, configStream);
+            TreeMap<EntityType, List<Entity>> entityMap = getEntityTypeListMap(extensionName, jobName, configStream);
```

Comment:
Same comment as above for the declarations in this class too. Method signatures and declaration should just have Map.